config/chime_science_run_gpu_21cm.yaml

```diff
@@ -0,0 +1,3596 @@
+##########################################
+#
+# chime_science_run_gpu.yaml
+#
+# Config to support the CHIME project with 2048 elements, and 1024 frequencies.
+#
+# This config contains the following main components:
+#
+# - Packet capture from the ICE boards including FPGA flag handling and normalization
+# - N^2 kernels and output for various data projects to receiver systems
+# - FRB FFT Fan beam and packet Tx
+# - Pulsar 10 coherent beam system abd packet Tx
+# - RFI detection and summary stats system (only reports on RFI, no normalization yet)
```

Comment:
We should just merge the changes for the absorber config directly into the main `science_run` and `no_output` configs at this point.

Reply:
Also add a comment in here about the absorber system